Cargo.toml

```diff
@@ -23,6 +26,7 @@ url = "2.1.0"
 hyper-native-tls = {version = "0.3.0", optional=true}
 mime_guess = "2.0"
 open = "1"
+openssl = { version = "*", features = ["vendored"] }
```

Comment:
Please stick the `openssl` version number.

Reply:
Done